planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/occluded_crosswalk.hpp

```diff
@@ -80,7 +80,7 @@ double calculate_detection_range(
 /// @return selected and inflated objects
 std::vector<autoware_perception_msgs::msg::PredictedObject> select_and_inflate_objects(
   const std::vector<autoware_perception_msgs::msg::PredictedObject> & objects,
-  const double velocity_threshold, const bool skip_pedestrians, const double inflate_size);
+  const double & velocity_threshold, const bool skip_pedestrians, const double inflate_size);
```

Comment:
In general for literal type like int/double/etc,  passing by value is faster than passing by reference. But it is OK

Reply:
Thanks for the advice.

Reply:
I reverted this change.

Reply:
I agree with @soblin 's comment. I created a suggestion.